### DISCUSSION TITLE
Add support for tvOS

### DISCUSSION THREAD

**Discussion by ericlifs** [mmazzarolo/react-native-universal-monorepo#8] [Ideas]
I think the title is self explanatory, I want to propose adding support for react-native-tvOS in this awesome repo.

Some of the caveats I found while working with this:
* There are some specific tvOS types for react-native (https://gist.github.com/Krisztiaan/11856c0faffc70f00766b7f791375a75)
* The react-native library needs to be installed from ("react-native": "npm:react-native-tvos@latest")

Thank you very much!

**Comment by mmazzarolo** [comment]
I like the idea of supporting tvOS 👍 
The only thing stopping me from doing it is that I have 0 experience with it and I don't know how easy it would be to test it on my machine (but I assume it should be doable without issues on macOS, right?). 
I'm a bit busy writing the blog posts (+ work) these days, but I'm open to help with PR reviews and chat 👍

**Reply by ericlifs** [reply]
Yeah, it's pretty straightforward. It works the same way as iOS does. In my case, we are running the metro bundler through the terminal and then playing the play button on xcode, but you can do all together as well from the terminal.

**Comment by mmazzarolo** [comment]
#15 tvOS and Android TV are now supported
